integration_tests/src/metric.js

```diff
@@ -0,0 +1,31 @@
+const tracer = require("dd-trace").init({
+  flushInterval: 0,
+  logLevel: "debug",
+});
+
+const { datadog, sendDistributionMetric } = require("datadog-lambda-js");
+
+let invocationCount = 0;
+
+async function myHandler(event, context) {
+  sendDistributionMetric("serverless.lambda-extension.integration-test.count", invocationCount);
+  invocationCount += 1;
+  return {
+    statusCode: 200,
+    body: 'ok'
+  };
+}
+
+async function myTimeoutHandler(event, context) {
+  sendDistributionMetric("serverless.lambda-extension.integration-test.count", invocationCount);
+  await new Promise(r => setTimeout(r, 30*1000)); //30 sec to be sure 
```

Comment:
Nit: For testing timeouts, I prefer to create an infinite loop. That way, the function  will definitely time out no matter how I adjust the Lambda timeout setting.

Reply:
I've adjust it with the max AWS allowed timeout value ✅